rplugin/node/vim-package-info/parser.js

```diff
@@ -3,16 +3,18 @@ const toml = require("toml");
 // Format:
 // [ [start, end], [start, end] ]
 const depMarkers = {
-  "package.json": [[/["|'](dependencies)["|']/, /\}/], [/["|'](devDependencies)["|']/, /\}/]],
-  "Cargo.toml": [[/\[(.*dependencies)\]/, /^ *\[.*\].*/]],
-  "requirements.txt": null,
-  "Pipfile": [[/\[(packages)\]/, /^ *\[.*\].*/], [/\[(dev-packages)\]/, /^ *\[.*\].*/]]
+  "javascript": [[/["|'](dependencies)["|']/, /\}/], [/["|'](devDependencies)["|']/, /\}/]],
+  "rust": [[/\[(.*dependencies)\]/, /^ *\[.*\].*/]],
+  "python:requirements": null,
+  "python:pipfile": [[/\[(packages)\]/, /^ *\[.*\].*/], [/\[(dev-packages)\]/, /^ *\[.*\].*/]],
+  "python:pyproject": [[/\[(.*dependencies)\]/, /^ *\[.*\].*/]]
 };
 const nameParserRegex = {
-  "package.json": /['|"](.*)['|"] *:/,
-  "Cargo.toml": /([a-zA-Z0-9\-_]*) *=.*/,
-  "requirements.txt": /^ *([a-zA-Z_]+[a-zA-Z0-9\-_]*).*/,
-  "Pipfile": /"?([a-zA-Z0-9\-_]*)"? *=.*/
+  "javascript": /['|"](.*)['|"] *:/,
+  "rust": /([a-zA-Z0-9\-_]*) *=.*/,
+  "python:requirements": /^ *([a-zA-Z_]+[a-zA-Z0-9\-_]*).*/,
+  "python:pipfile": /"?([a-zA-Z0-9\-_]*)"? *=.*/,
+  "python:pyproject": /([a-zA-Z0-9\-_]*) *=.*/
```

Comment:
Have to handle when the dependency has `"` arround it.
```suggestion
  "python:pyproject": /['|"]?([a-zA-Z0-9\-_]*)['|"]? *=.*/
```

Reply:
Could you also add a test case for this one.